phpcs-ruleset.xml

```diff
@@ -7,11 +7,7 @@
     <!-- Minified files don't have to comply with coding standards. -->
     <exclude-pattern>*.min.css</exclude-pattern>
     <exclude-pattern>*.min.js</exclude-pattern>
-    <exclude-pattern>profiles/common/themes/ec_resp/scripts/ec.js</exclude-pattern>
     <exclude-pattern>profiles/common/themes/ec_resp/css/ec_resp.css</exclude-pattern>
```

Comment:
This line concerns ec_resp, not ec_resp17 (I'm wainting for another merge to avoid some conflicts to rename ec_resp.css in ec_resp.min.css).
I reverted the wrong commit about this.